src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -532,21 +532,31 @@ func (b *ByocAws) Follow(ctx context.Context, req *defangv1.TailRequest) (client
 		etag = "" // no need to filter by etag
 		stopWhenCDTaskDone = true
 	} else {
-		// Tail CD, kaniko, and all services (this requires ProjectName to be set)
-		kanikoTail := ecs.LogGroupInput{LogGroupARN: b.driver.MakeARN("logs", "log-group:"+b.stackDir("builds"))} // must match logic in ecs/common.ts
-		term.Debug("Tailing kaniko logs", kanikoTail.LogGroupARN)
-		servicesTail := ecs.LogGroupInput{LogGroupARN: b.driver.MakeARN("logs", "log-group:"+b.stackDir("logs"))} // must match logic in ecs/common.ts
-		term.Debug("Tailing services logs", servicesTail.LogGroupARN)
-		ecsTail := ecs.LogGroupInput{LogGroupARN: b.driver.MakeARN("logs", "log-group:"+b.stackDir("ecs"))} // must match logic in ecs/common.ts
-		term.Debug("Tailing ecs events logs", ecsTail.LogGroupARN)
-		cdTail := ecs.LogGroupInput{LogGroupARN: b.driver.LogGroupARN}
-		taskArn = b.cdTasks[etag]
-		if taskArn != nil {
-			// If we know the CD task ARN, only tail the logstream for the CD task
-			cdTail.LogStreamNames = []string{ecs.GetLogStreamForTaskID(ecs.GetTaskID(taskArn))}
+		groups := make([]ecs.LogGroupInput, 0, 4)
+		// tail build or service logs (this requires ProjectName to be set)
+		if req.Build {
```

Comment:
Maybe this should be flags / multiple boolean? Often I do want to see everything: cd -> build -> runtime

Reply:
Updated in https://github.com/DefangLabs/defang/pull/769